### PR TITLE
feat: add new command to get text

### PR DESCRIPTION
**Pull request by tandt53**
Hi @Alpaca00 ,

Thanks for your application and solution with `dragAndDropWithCommandExtension`. It gave me a hint to fix the issue when getting text from Text widget. 
Now I want to add a new command `getTextWithCommandExtension` to get text from Text widget that contains TextSpan widgets as children. 

Hope that it will help other like your code did to me. 

I added
- 1 widget in the body of main.
- 1 file lib/get_text_command.dart with helper methods to get locator, find element and get the text.
It works on my local.

https://github.com/user-attachments/assets/f6c2d5b6-334e-45c5-a5c3-72f2731c5a09

### CODE REVIEW

**Comment by Alpaca00** [comment]
I'm glad the solution was helpful! Your extension for handling `TextSpan` widgets is a great addition - surely it will benefit others too.

File: lib/get_text_command.dart [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    // Get the widget element
+    final Element element = prober.element(finder);
+
+    // Get text data from Text widget
+    String textData = '';
+
+    if (element.widget is Text) {
+      textData = getTextFromWidget(element.widget as Text);
+    } else {
+      textData = 'Found element is not a Text widget';
```
Comment (Alpaca00) [inline lib/get_text_command.dart]: Returning an error in the response for the user could be a potential improvement, making the behavior clearer and more understandable for others.

Example code:
```
      return const GetTextResult(
        false,
        data: {
          'errorCode': 'NOT_A_TEXT_WIDGET',
          'error': 'Found element is not a Text widget'
        },
      );
```
Response if the request is incorrect:
```
{'isError': false, 'response': {'success': false, 'errorCode': 'NOT_A_TEXT_WIDGET', 'error': 'Found element is not a Text widget'}, 'type': '_extensionType', 'method': 'ext.flutter.driver'}
```

At the moment, we are receiving this:
```
{'isError': false, 'response': {'success': true, 'text': 'Found element is not a Text widget'}, ...}
```

Comment (tandt53) [reply lib/get_text_command.dart]: nice. I updated the code handling get text error, and rename the drag command as your suggestion [here](https://github.com/appium/appium-flutter-driver/pull/755#discussion_r1864802802)

Comment (Alpaca00) [reply lib/get_text_command.dart]: looks great, thank you so much!

**Pull request merged**